scripts/test_no_default_features.sh

```diff
@@ -6,7 +6,6 @@ trust_dns_dir=$(dirname $0)/..
 cd ${trust_dns_dir:?}
 
 # Build all tests
-cargo test --manifest-path crates/proto/Cargo.toml --no-default-features
```

Comment:
I think in the long run we really want this to run, but we'd feature flag off the code that is no longer included (which might be a lot).

Reply:
Rather than removing this entirely, how about we have `--no-default-features --feature=tokio`?

That way the tests are the same as before, just slightly inaccurately named. It should fix the test failure from before.

Reply:
Good call, done.

Also fixed the tests by adding `--features tokio` to the resolver line.